src/utils/api.ts

```diff
@@ -0,0 +1,7 @@
+export function getMiddlewareUrl() {
+	return 'http://localhost:8000';
```

Comment:
I still need to look if env variables are supported, if not I'll prob do a `__DEV__` check and hardcode localhost and default to production middleware url.